PullRequestCreated/PullRequestCreated.ts

```diff
@@ -40,12 +40,8 @@ class PullRequestCreated extends OctokitAction {
         }
       }
     } else {
-      const mentionedIssues = this.findMentionedIssues(pr);
-      const notMentionedIssues = linkedIssues.filter(x => !mentionedIssues.has(x));
-      console.log(`Adding the following ticket in description: ${notMentionedIssues}`);
-      if (notMentionedIssues.length > 0) {
```

Comment:
The `if` was removed because `.match` can never return 0 issues. It will either return `null` for 0 matches, or an array with one or more items.